arex-instrumentation/database/arex-database-mybatis3/src/main/java/io/arex/inst/database/mybatis3/InternalExecutor.java

```diff
@@ -255,4 +260,20 @@ public static DatabaseExtractor createExtractor(MappedStatement mappedStatement,
         }
         return new DatabaseExtractor(originalSql, Serializer.serialize(parameters, ArexConstants.JACKSON_REQUEST_SERIALIZER), methodName);
     }
+
+    private static Reflector newReflector(Class<?> clazz) {
+        try {
+            return new Reflector(clazz);
+        } catch (IllegalAccessError e) {
+            // compatible mybatis version < 3.3.0 , use private constructor
+            try {
+                Constructor<Reflector> constructor = Reflector.class.getDeclaredConstructor(Class.class);
```

Comment:
mabatis version 
```java
> 3.3.0: org.apache.ibatis.reflection.DefaultReflectorFactory
< 3.3.0: org.apache.ibatis.reflection.Reflector#forClass

Reflection utils
io.arex.agent.bootstrap.util.ReflectUtil
io.arex.inst.runtime.util.fastreflect.MethodHolder#invoke
```

Reply:
Here we still use the constructor to create a reflector instance of and maintain a cache storage this instance. The reason is that there is no DefaultReflectorFactory in earlier versions, and an error will be reported when the class is loaded